setup.py

```diff
@@ -34,7 +34,6 @@
         "sphinxcontrib-httpdomain>=1.5.0",
         "sphinxcontrib-openapi>=0.3.0",
         "sphinxcontrib-redoc>=1.5.1",
-        "mistune<2.0.0",
```

Comment:
Perhaps we should also upgrade the lower version of `sphinxcontrib-openapi` to 0.8.0 or higher, so that it'll be fully clear that the mistune pin can be removed.

Reply:
Also, it would be good to invert the two commits, i.e. having mistune fix come first and the volume mounting second, so that both commits of the branch would pass the CI.

Reply:
Strictly speaking, it is only necessary there where we pinned `mistune` dependency, i.e. only here. Because the other repositories are working nicely with older versions too.

That said, it may be nice indeed to update others, for consistency's sake, since we are touching all the repositories?